data/hai/hai reveal 6 end.txt

```diff
@@ -0,0 +1,672 @@
+			`	After about thirty minutes of looking around a young boy walks up and addresses you, "Hey stranger sir, I've been asked to tell you that Terry is out on the bluff waiting for you."`
+			`	Before you can even respond he's dashed away.`
+			choice
+				`	(Go to the bluff.)`
+			`	When you get out to the bluff there is a single individual standing on a level section with scrubby grass tufting between bare rock. The city is obscured from here, and you could believe you were isolated from all of civilization. Upon closer inspection the individual does indeed turn out to be Terry. The wind whips her hair around her constantly and there is a chill here that is only partly the fault of the wind. All of this trouble, all of this death and suffering, everything that was risked between the Hai and humanity. Could this one young woman really have been behind all of it?`
+			`	As you approach she turns and you can see she has been crying. "You know you ruined everything," she says, "we were so close."`
+			`	The wind almost steals her last word and you unconsciously step closer to hear her better, which causes her to tense.`
+			choice
+				`	"So close to what?"`
+			`	"To changing everything, to remaking the Syndicate, to creating a whole new political order."`
+			`	She wipes away a tear and visibly steels herself to speak, "I was Alastair's chosen successor, I had already dealt with the competition, a few more years and I'd have been a hero!"`
+			`	She turns slightly and gestures at the sea. "Instead now I'm here, and you get to be the hero, a dumb war hero," her voice rises in pitch and volume, "Never mind a brighter future for humanity!"`
+			`	Her expression seems to be wrestling between fury and despair at this point, "So now what? You here to kill me then?!"`
+			choice
+				`	"You kidnapped thousands of people, how is that a brighter future?"`
+			`	"Of course you have no idea!" she throws up her hands and expresses a noise of frustration.`
+			`	Her voice drops to a more reasonable level. "I never wanted to kidnap people," and she gets quieter again, "I inherited that legacy..."`
+			`	"You still profited off it, did nothing to stop it," you say.`
+			`	Your tone had not been accusatory, but merely matter of fact, and yet they seem to have struck her like a sack of bricks. Her composure shatters and she begins to sob, sinking to her knees and shielding her face from the wind.`
+			`	"It was never meant to be like this," she forces out, "but the Philosophers were our only resource, and we weren't in control anymore."`
+			`	"We wanted to reform the government, make it better, make it work for the people instead of against them. We dealt with the devil to keep that dream alive I suppose. I guess I should have expected it to get us burnt eventually."`
```

Comment:
```suggestion
			`	"Of course you have no idea!" she throws up her hands and expresses a noise of frustration.`
			`	Her voice drops to a more reasonable level. "I never wanted to kidnap people," and she gets quieter again, "I inherited that legacy..."`
			`	"You still profited off it, did nothing to stop it," you point out.`
			`	Your tone had not been accusatory, but merely matter of fact, and yet they seem to have struck her like a sack of bricks. Her composure shatters, and she begins to sob, sinking to her knees and shielding her face from the wind.`
			`	"It was never meant to be like this," she forces out between sobs. "But the Philosophers were our only resource, and we weren't in control anymore.`
			`	"We wanted to reform the government, make it better, make it work for the people instead of against them. We dealt with the devil to keep that dream alive. I guess I should have expected it to get us burnt eventually."`
```
Alternative suggestion.

Reply:
I'd like to work on that interjection to the dialogue (`and she gets quieter again`) if we keep it in. How about this?
```suggestion
			`	"Of course you have no idea!" she throws up her hands and expresses a noise of frustration.`
			`	Her voice drops to a more reasonable level. "I never wanted to kidnap people," Terry says, then lowers her voice even more. "I inherited that legacy..."`
			`	"You still profited off it, did nothing to stop it," you point out.`
			`	Your tone had not been accusatory, but merely matter of fact, and yet they seem to have struck her like a sack of bricks. Her composure shatters, and she begins to sob, sinking to her knees and shielding her face from the wind.`
			`	"It was never meant to be like this," she forces out between sobs. "But the Philosophers were our only resource, and we weren't in control anymore.`
			`	"We wanted to reform the government, make it better, make it work for the people instead of against them. We dealt with the devil to keep that dream alive. I guess I should have expected it to get us burnt eventually."`
```